app/routes/file.js

```diff
@@ -109,15 +116,28 @@ const FileRoute = function(app, headerDB, updateDuration) {
     console.info("File exists in cache. Not downloading", fileUrl, new Date());
   }
 
-  function sendResponse(res, fileUrl) {
-    res.status(202)
-      .send({
-        status: 202,
-        message: "File is downloading"
-      });
+  function sendInvalidResponseResponse(res, fileUrl, statusCode) {
```

Comment:
@stulees I meant that. It means this function will send a response to the client regarding an invalid response from the host server. cheers